go.mod

```diff
@@ -9,6 +9,8 @@ require (
 	github.com/aymerick/raymond v2.0.2+incompatible
 	github.com/boumenot/gocover-cobertura v1.2.0
 	github.com/cespare/xxhash/v2 v2.2.0
+	github.com/dustin/go-humanize v1.0.0
+	github.com/elastic/elastic-integration-corpus-generator-tool v0.3.1-0.20230126015529-f42e909f4536
```

Comment:
a few changes are needed in github.com/elastic/elastic-integration-corpus-generator-tool before we can tag a proper version to use a required package here